debug_module.tex

```diff
@@ -363,4 +363,8 @@ \section{Debug Module DMI Registers} \label{dmdebbus}
 When read, unimplemented Debug Module DMI Registers return 0. Writing them has
 no effect.
 
+For each register it is possible to check whether it is implemented by checking
+that it's not 0 (eg. \Rsbcs), or by checking bits in another register (eg.
```

Comment:
maybe clarify that you just have to read it "it's not 0 -> reading it returns 0"